src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -176,7 +176,14 @@
 K8s_Cluster_Info = "k8s_cluster_info.txt"
 Outbound_Network_Connectivity_Check = "outbound_network_connectivity_check.txt"
 Events_of_Incomplete_Diagnoser_Job = "diagnoser_failure_events.txt"
-
+# Connect Precheck Diagnoser constants
+Pre_Onboarding_Inspector_Job_Registry_Path = "connectprecheck.azurecr.io/helm/pre-onboarding-inspector:0.1.0"
```

Comment:
Create a new helm chart referencing multi-architecture image: arck8sdiagnoser.azurecr.io/public/preonboardjobscript:v0.1.1 and update the constant with the newer helm chart.